sia/components/renter.go

```diff
@@ -1,14 +1,6 @@
 package components
 
-import (
-// "github.com/NebulousLabs/Sia/consensus"
-)
-
-type RenterUpdate struct {
-	HostDB HostDB
-}
-
 type Renter interface {
-	// UpdateRenter changes the settings used by the host.
-	UpdateRenter(RenterUpdate) error
+	RenameFile(currentName, newName string) error
+	RentFile(filename, nickname string, totalPieces, requiredPieces, optimalRecoveryPieces int) error
```

Comment:
a struct might be nicer here...it feels kinda like overkill, but on the other hand it lets the caller write:
```go
r.RentFile(components.File{
    Name: "foo",
    Nickname: "bar",
    TotalPieces: 1,
    RequiredPieces: 2,
    OptimalRecPieces: 3,
})
```
instead of:
```go
r.RentFile("foo", "bar", 1, 2, 3) // what do these numbers mean??
```